libs/service-portal/licenses/src/lib/messages.ts

```diff
@@ -21,7 +25,46 @@ export const m = defineMessages({
   drivingLicenseDescription: {
     id: 'sp.license:driving-license-description',
     defaultMessage:
-      'Hér birtast upplýsingar um ökuskírteini þitt ásam þeim ökuréttindum sem þú ert með í gildi á hverjum tíma.',
+      'Hér birtast upplýsingar um ökuskírteini þitt ásamt þeim ökuréttindum sem þú ert með í gildi á hverjum tíma.',
+  },
+  ADRLicense: {
+    id: 'sp.license:adr-license',
+    defaultMessage: 'ADR réttindi',
+  },
+  yourADRLicense: {
+    id: 'sp.license:your-adr-license',
+    defaultMessage: 'ADR réttindin þín',
+  },
+
+  adrLicenseDescription: {
+    id: 'sp.license:adr-license-description',
+    defaultMessage:
+      'Hér birtast upplýsingar um ADR skírteini þitt ásamt þeim réttindum sem þú ert með í gildi á hverjum tíma.',
+  },
+  huntingLicense: {
+    id: 'sp.license:hunting-license',
```

Comment:
Change `hunting` to `firearm` since a hunting license will come later (veiðileyfi)

Reply:
Done